ansible/deploy-proteus-events.yml

```diff
@@ -0,0 +1,27 @@
+- hosts: events.proteus.test.ooni.io
+  vars:
+    proteus_events_https_port: 443
+    letsencrypt_nginx: yes
+    letsencrypt_domains: "{{ inventory_hostname }}"
+    proteus_database_url: "postgres://proteus:{{ proteus_database_password_testing }}@db-1.proteus.test.ooni.io:5432/proteus?sslmode=require"
```

Comment:
`sslmode=require` is nice, but it's using `/etc/ssl/certs/ssl-cert-snakeoil.`, so what's the point?